### PR TITLE
POC WIP DNM script to bump charts

### PR DESCRIPTION
**Pull request by faiq**
**What type of PR is this?**

Automation 

**What this PR does/ why we need it**:

accompanying changes here https://github.com/mesosphere/dispatch-tasks/pull/27

loop over all charts, attempt to bump

this aims to create a pr per dependency bump per chart. the goal would be to capture changes like https://github.com/mesosphere/charts/pull/985 automatically on a pulling interval. We can set this to be every 30 minutes. 

The tool attempts to create a branch per chart per dependency bump. It checks out and pushes a predictable branch name.

We can also run this for particular charts we own -- kommander comes to mind.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Checklist**

* [ ] *If a chart is changed, the chart version is correctly incremented.*
* [ ] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

File: bump.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -euo pipefail
+cd $1
+find * -prune -type d | while IFS= read -r d; do
+  echo "attempting to bump $d"
+  cd $d
+  if [ -f requirements.yaml ]; then
+    bump_requirements --user=mesosphere --project=charts --bump-option=sliced --dependencies=*;
```
Comment (orsenthil) [inline bump.sh:8]: - Is attempting to bump every requirements.yaml the right thing to do?
-  What might trigger this? A cron?

File: bump.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```
Comment (makkes) [inline bump.sh:1]: Please add a comment explaining the purpose of the script and how it is supposed to be called here for teammates to get a quick understanding of this script.

**Review by makkes — CHANGES_REQUESTED** [review]

File: bump.sh (hunk 2 above)
Comment (faiq) [reply bump.sh:1]: You got it -- ill address next pass on this PR

**Comment by faiq** [comment]
im closing this until i can do this again

**Pull request closed without merging**